src/System.Private.CoreLib/src/Internal/Runtime/InteropServices/IsolatedComponentLoadContext.cs

```diff
@@ -17,7 +17,7 @@ internal class IsolatedComponentLoadContext : AssemblyLoadContext
     {
         private readonly AssemblyDependencyResolver _resolver;
 
-        public IsolatedComponentLoadContext(string componentAssemblyPath)
+        public IsolatedComponentLoadContext(string componentAssemblyPath) : Base($"IsolatedComponentLoadContext({componentAssemblyPath})")
```

Comment:
I thought it has to be `base` (lowercases)

Reply:
you are right `base` is the keyword